src/block/block_ext.c

```diff
@@ -74,11 +79,20 @@ __block_off_srch(WT_EXT **head, wt_off_t off, WT_EXT ***stack, bool skip_off)
      * the skip_off flag is set, offset the skiplist array by the depth specified in this particular
      * structure.
      */
-    for (i = WT_SKIP_MAXDEPTH - 1, extp = &head[i]; i >= 0;)
-        if (*extp != NULL && (*extp)->off < off)
+    ext_tmp = NULL;
+    for (i = WT_SKIP_MAXDEPTH - 1, extp = &head[i]; i >= 0;) {
+        if (*extp != NULL && (*extp)->off < off) {
+            ext_tmp = *extp;
             extp = &(*extp)->next[i + (skip_off ? (*extp)->depth : 0)];
-        else
+        } else {
             stack[i--] = extp--;
+        }
```

Comment:
```suggestion
        } else
            stack[i--] = extp--;
```
We don't use `{` `}` when it's a one-line statement.

Reply:
done